src/ggml.c

```diff
@@ -16670,11 +16668,11 @@ void ggml_graph_export(const struct ggml_cgraph * cgraph, const char * fname) {
                 {
                     struct ggml_tensor * args[2 + GGML_MAX_OPT] = { NULL };
 
-                    args[0] = tensor->src0;
-                    args[1] = tensor->src1;
+                    args[0] = tensor->src[0];
+                    args[1] = tensor->src[1];
 
                     for (int j = 0; j < GGML_MAX_OPT; ++j) {
-                        args[2 + j] = tensor->opt[j];
+                        args[2 + j] = tensor->src[j];
                     }
```

Comment:
Again this is broken. You need to reconsider all the code that looked at `src0`, `src1` and `opt` and replace it with a single loop that iterates over src.

Reply:
> Again this is broken. You need to reconsider all the code that looked at `src0`, `src1` and `opt` and replace it with a single loop that iterates over `src`.

What's the best approach to fix these kinds, `j = 2`?

Reply:
> > Again this is broken. You need to reconsider all the code that looked at `src0`, `src1` and `opt` and replace it with a single loop that iterates over `src`.
> 
> What's the best approach to fix these kinds, `j = 2`?

Wait I'm dumb, fixing this :)

Reply:
I don't think there is a simple solution that you can apply everywhere. You have to consider it on a case-by-case basis. In this code for example, the best solution may be to remove `args` and just use `src` directly.